tests/fsharp/core/fsfromfsviacs/lib2.cs

```diff
@@ -32,6 +32,10 @@ public class Lib2
     public static FSharpRef<int> ri1 = new FSharpRef<int>(3);
```

Comment:
@TIHan You might want to add some struct-record testing to this fsfromfsviacs test.  It both checks the access of F# code from C#, and checks the access of struct records across assembly boundaries.  This is important because fields can be accessed directly within an assembly, but across assemblies they can't be and values must be accessed via helper methods instead.